mkdocs.yml

```
@@ -58,14 +58,34 @@ nav:
         Gateway: api-types/gateway.md
         HTTPRoute: api-types/httproute.md
         ReferenceGrant: api-types/referencegrant.md
-    - API specification: references/spec.md
+    - API Specification: references/spec.md
     - Policy Attachment: references/policy-attachment.md
     - Enhancement Proposals:
-      - Overview: contributing/gep.md
-      - ... | geps/gep-*.md
+      - Overview: geps/overview.md
+      - Provisional:
+        - geps/gep-735.md
+        - geps/gep-1282.md
+        - geps/gep-1324.md
+      - Implementable:
+        - geps/gep-1364.md
+      - Experimental:
+        - geps/gep-709.md
+        - geps/gep-713.md
+        - geps/gep-726.md
+        - geps/gep-957.md
+        - geps/gep-1016.md
+        - geps/gep-1323.md
+      - Standard:
+        - geps/gep-718.md
+        - geps/gep-724.md
+        - geps/gep-746.md
+        - geps/gep-820.md
+        - geps/gep-851.md
+        - geps/gep-917.md
+        - geps/gep-922.md
```

Could we sort these in reverse chronological order so the newest/most relevant GEPs appear first?
```suggestion
      - Provisional:
        - geps/gep-1324.md
        - geps/gep-1282.md
        - geps/gep-735.md
      - Implementable:
        - geps/gep-1364.md
      - Experimental:
        - geps/gep-1323.md
        - geps/gep-1016.md
        - geps/gep-957.md
        - geps/gep-726.md
        - geps/gep-713.md
        - geps/gep-709.md
      - Standard:
        - geps/gep-922.md
        - geps/gep-917.md
        - geps/gep-851.md
        - geps/gep-820.md
        - geps/gep-746.md
        - geps/gep-724.md
        - geps/gep-718.md
```

---

Updated, thanks!